# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/gl/GLContextProviderImpl.h
# Commit: dd00e2da3a0f
# Full Hash: dd00e2da3a0f11a7cc0af9e5ece21a339f4393eb
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-11 04:04:33
# Regressor Bug: 1632249
# File Overlap Count: 1
# Description:
#   Bug 1632249 - Support compositing out-of-process WebGL. r=handyman,lsalzman,nical,geckoview-reviewers,agi,imanol
#   
#   * Majorly simplity CanvasRenderer
#   * Replace GLScreenBuffer with trivial GLSwapChain
#   * Use descriptor structs so that future SharedSurface changes aren't so painful
# ==============================================================================

diff -r 69cc1de6f754 -r dd00e2da3a0f gfx/gl/GLContextProviderImpl.h
--- a/gfx/gl/GLContextProviderImpl.h	Wed Jun 10 21:57:19 2020 +0000
+++ b/gfx/gl/GLContextProviderImpl.h	Wed Jun 10 22:21:02 2020 +0000
@@ -66,12 +66,12 @@
    * @return Context to use for offscreen rendering
    */
   static already_AddRefed<GLContext> CreateOffscreen(
-      const mozilla::gfx::IntSize& size, const SurfaceCaps& minCaps,
-      CreateContextFlags flags, nsACString* const out_failureId);
+      const mozilla::gfx::IntSize& size, const GLContextCreateDesc&,
+      nsACString* const out_failureId);
 
   // Just create a context. We'll add offscreen stuff ourselves.
   static already_AddRefed<GLContext> CreateHeadless(
-      CreateContextFlags flags, nsACString* const out_failureId);
+      const GLContextCreateDesc&, nsACString* const out_failureId);
 
   /**
    * Create wrapping Gecko GLContext for external gl context.